CoinDock-api/app/Models/V1/RecoveryKey.php

```diff
@@ -58,6 +58,26 @@ public function store(User $user)
         return $recoveryGeneration;
     }
 
+
+
+    //Reg-generarion of recovery-codes
+    public function reGenerateRecoveryKeys(User $user){
+        
+        $recovery = RecoveryKey::whereUserId($user->id)->first();
+        //randomizing the dictionary words
```

Comment:
regenerating means Are we updating the recovery keys?

Reply:
If Yes, then what is the meaning for maintaining status in the recovery_keys table.